src/databricks/labs/ucx/framework/crawlers.py

```diff
@@ -231,6 +233,28 @@ def _full_name(self) -> str:
         """
         return f"{self._catalog}.{self._schema}.{self._table}"
 
+    @staticmethod
```

Comment:
create a function of it and call it `escape_sql_identifier`, it doesn't really belong to this class, as it turns out from number of `CrawlerBase.` prefixes

Reply:
Yeah, that makes sense, I was about to ask that. Where do we put functions? A new Python source in the root of the project? Or some `utils` subdir?